### PR TITLE
sk/fixbigdecomp 0.4

### PR DESCRIPTION
**Pull request by StefanKarpinski**

### CODE REVIEW

File: test/bigfloat.jl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@test big(2)^big(3) == 8
+
+for T in [Int8, UInt8, Int16, UInt16, Int32, UInt32, Int64, UInt64, Int128, UInt128, BigInt]
```
Comment (tkelman) [inline test/bigfloat.jl]: bigint tests in the bigfloat file?

**Comment by StefanKarpinski** [comment]
@tkelman, @vtjnash: I snuck in a cherry-pick of the dsymutil fix since it was needed.

File: test/bigfloat.jl (hunk 1 above)
Comment (StefanKarpinski) [inline test/bigfloat.jl]: Ah, I meant to test raising `big(2.0)` to various powers.

File: test/bigfloat.jl [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	@test T(2)^big(3) == 8
```
Comment (tkelman) [inline test/bigfloat.jl]: spaces, not tab

Comment (StefanKarpinski) [inline test/bigfloat.jl]: I already merged the original, so I'm going to keep the cherry-pick as close as possible and also pick the fixup, which also fixes the tests to actually test BigFloats.

**Pull request merged**